credits.txt

```diff
@@ -11,6 +11,8 @@ Photos from Wikimedia Commons
   Matthew Field (CC-BY-SA 3.0)
 and other photographers
   Chris Pickel (CC-BY-SA 4.0)
+  Michael Wilson (CC-BY-4.0)
+  @Gardevou (CC-BY-4.0)
```

Comment:
It's the name emotion wants to go under. I can't make him use his real name. (Yes, it is the same as the one from discord.